dev/ci/user-overlays/14059-ejgallego-coq+path_boot.sh

```diff
@@ -0,0 +1 @@
+overlay elpi https://github.com/ejgallego/coq-elpi coq+path_boot 14059
```

Comment:
@ejgallego where's the pr for this?

Reply:
Ouch, let me open it right now.

Reply:
https://github.com/LPCIC/coq-elpi/pull/287